src/pages/home/sidebar/AvatarWithDelegateAvatar.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Avatar from '@components/Avatar';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as UserUtils from '@libs/UserUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ProfileAvatarWithIndicator from './ProfileAvatarWithIndicator';
+
+type AvatarWithDelegateAvatarProps = {
+    /** Emoji status */
```

Comment:
```suggestion
    /** original account of delegate */
```

Reply:
I think I won't be able to merge if I do that unfortunately, a little wonky but @allgandalf can you add it into your PR? I'll go ahead and merge this as is.